testing/src/main/java/org/oppia/android/testing/lightweightcheckpointing/ExplorationCheckpointTestHelper.kt

```diff
@@ -13,23 +13,37 @@ import org.mockito.MockitoAnnotations
 import org.oppia.android.app.model.ExplorationCheckpoint
 import org.oppia.android.app.model.ProfileId
 import org.oppia.android.domain.exploration.lightweightcheckpointing.ExplorationCheckpointController
+import org.oppia.android.domain.topic.FRACTIONS_EXPLORATION_ID_0
+import org.oppia.android.domain.topic.FRACTIONS_EXPLORATION_ID_1
+import org.oppia.android.domain.topic.RATIOS_EXPLORATION_ID_0
 import org.oppia.android.testing.threading.TestCoroutineDispatchers
+import org.oppia.android.testing.time.FakeOppiaClock
 import org.oppia.android.util.data.AsyncResult
 import org.oppia.android.util.data.DataProvider
 import org.oppia.android.util.data.DataProviders.Companion.toLiveData
 import javax.inject.Inject
 import javax.inject.Singleton
 
-const val FAKE_EXPLORATION_ID_1 = "fake_exploration_id_1"
-const val FAKE_EXPLORATION_TITLE_1 = "Fake exploration title_1"
-const val FAKE_EXPLORATION_ID_2 = "fake_exploration_id_2"
-const val FAKE_EXPLORATION_TITLE_2 = "Fake exploration title_2"
+const val FRACTIONS_EXPLORATION_0_TITLE = "What is a Fraction?"
```

Comment:
Please add KDocs for all currently undocumented non-private members in this class & remove the exemption from scripts/assets/kdoc_validity_exemptions.textproto.

Reply:
done